.github/workflows/coding-standards.yml

```diff
@@ -0,0 +1,59 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: "Check Coding Standards"
+
+on:
+    pull_request:
+    push:
+        branches:
+            - "master"
+
+jobs:
+    coding-standards:
+        name: "Check Coding Standards"
+
+        runs-on: ${{ matrix.operating-system }}
+
+        strategy:
+            matrix:
+                dependencies:
+                    - "locked"
+                php-version:
+                    - "7.4"
+                operating-system:
+                    - "ubuntu-latest"
+
+        steps:
+            - name: "Checkout"
+              uses: "actions/checkout@v2"
+
+            - name: "Install PHP"
+              uses: "shivammathur/setup-php@v2"
+              with:
+                  coverage: "pcov"
+                  php-version: "${{ matrix.php-version }}"
+                  ini-values: memory_limit=-1
+
+            - name: "Cache dependencies"
+              uses: "actions/cache@v2"
+              with:
+                  path: |
+                      ~/.composer/cache
+                      vendor
+                  key: "php-${{ matrix.php-version }}-${{ matrix.dependencies }}"
+                  restore-keys: "php-${{ matrix.php-version }}-${{ matrix.dependencies }}"
```

Comment:
Composer 2 without caching installs dependencies faster than either version of Composer with caching 😉  I recommend removing this and adding `tools: composer:v2` to the "Install PHP" step (and doing this across all the workflows)

Reply:
See https://github.com/thephpleague/skeleton/pull/152/commits/66a5b43c43db45698b90c7147c28f919bd3a22c6